src/cascadia/TerminalApp/AboutDialog.cpp

```diff
@@ -85,6 +85,10 @@ namespace winrt::TerminalApp::implementation
             co_await winrt::resume_after(std::chrono::seconds{ 3 });
             co_await wil::resume_foreground(strongThis->Dispatcher());
             UpdatesAvailable(true);
+#elif defined(WT_BRANDING_CANARY)
+
+            // Canary doesn't have a package context. Don't bother.
```

Comment:
It could -- we could allow the StoreContext check below to fail until we figure out what to do with it. Up to you!

Reply:
obv this code was written 15 minutes _before_ our meeting where we thought it might after all